cmd/trust-manager/app/app.go

```diff
@@ -113,6 +115,28 @@ func NewCommand() *cobra.Command {
 				return fmt.Errorf("failed to create manager: %w", err)
 			}
 
+			targetCache, err := cache.New(mgr.GetConfig(), cache.Options{
```

Comment:
We need a target cache because we only want to store metadata for the target resources.